ttnn/cpp/ttnn/operations/data_movement/transpose/device/transpose_op.cpp

```diff
@@ -54,7 +54,7 @@ void Transpose::validate(const std::vector<Tensor> &input_tensors) const {
     }
     if (this->dim == TransposeOpDim::HC) {
         if (row_major) {
-            TT_FATAL((W * input_tensor.element_size()) % ROW_MAJOR_STICK_WIDTH == 0, "Error");
+            TT_FATAL((W * input_tensor.element_size()) % input_tensor.buffer()->alignment() == 0, "Error");
```

Comment:
I see that this whole file just prints "error". I can't ask to fix it in this PR, but please check out this guide and uplift when you have a chance. Thanks!
https://github.com/tenstorrent/tt-metal/blob/main/ErrorMessageBestPractices.md

Reply:
I likely have another transpose change coming to improve perf on this kernel so I'll do it then